docs/user-flows.md

```diff
@@ -1,262 +1,157 @@
+Historically, Lighthouse has analyzed the cold pageload of a page. Starting in 2022 (Lighthouse v10), it can analyze and report on the entire page lifecycle via "user flows".
 
-> Is that checkout form after you add an item to your cart accessible?
+#### You might be interested in flows if…
 
-> What is the Cumulative Layout Shift of my SPA page transition?
+* … you want to run Lighthouse on your whole webapp, not just the landing page.
+* … you want to know if the checkout process is accessible.
+* … you want to know the Cumulative Layout Shift of my SPA page transition
 
-You want Lighthouse on a _flow_, not just a page load.
+In these cases, you want Lighthouse on a _flow_, not just a page load.
 
-This document describes how Lighthouse measures flows and offers recommendations on how to structure your own flow measurement with practical examples.
+## The three modes: Navigation, Timespan, Snapshot
 
-## Flow Building Blocks
+Lighthouse can now run in three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
 
-Flow measurement in Lighthouse is enabled by three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
+* <img src="https://user-images.githubusercontent.com/39191/168928225-f2157fda-5131-4bd0-9121-b1a0b2f869a7.png" height="80" align="middle"> **Navigation mode** analyzes a single page load. Prior to v10, all Lighthouse runs were essentially in this mode.
+* <img src="https://user-images.githubusercontent.com/39191/168928251-c7025cd5-0086-4db8-ae52-95a5b5675adf.png" height="80" align="middle"> **Timespan mode** analyzes an arbitrary period of time, typically containing user interactions.
+* <img src="https://user-images.githubusercontent.com/39191/168931653-b45e0b6b-c5bd-4d8d-85b6-fee8425186a0.png" height="80" align="middle"> **Snapshot mode** analyzes the page in a particular state.
 
-### Navigation
 
-Navigation reports analyze a single page load. Navigation is the most common type of report you'll see. In fact, all Lighthouse reports prior to v9 are navigation reports.
+|  |  |
+|---|---|
+| Navigation | <small> *Benefits* <br> ✅ Provides an overall performance score and all metrics.<br>✅ Contains the most advice of all report types (both time-based and state-based audits are available).<br> *Limitations* <br> 🤔 Cannot analyze form submissions or single page app transitions.<br>🤔 Cannot analyze content that isn't available immediately on page load.<br> *Use Cases* <br> 👍 Obtain a Lighthouse Performance score.<br>👍 Measure Performance metrics (First Contentful Paint, Largest Contentful Paint, Speed Index, Time to Interactive, Cumulative Layout Shift, Total Blocking Time).<br>👍 Assess Progressive Web App capabilities.</small> |
```

Comment:
If we're going to use a table here, can benefits/limitations/use cases each have there own column? If that doesn't look good then maybe we're better off using headings/subheadings for this.

Reply:
i tried the per-column look but there's just too much line-wrapping and it looked a lil whack.

(well.. what i had was slightly diff but the wrapping is similar to what it'd be:)
![image](https://user-images.githubusercontent.com/39191/169391787-365ff1a3-fb48-47f7-9fc2-c0fda38ecb2b.png)

Lemme try some clearer headings…

Reply:
yup better now
![image](https://user-images.githubusercontent.com/39191/169396931-1537d5e4-8056-414b-b017-39f39321ae10.png)